### PR TITLE
Some copy editing for README.md

### PR DESCRIPTION
**Pull request by CrackedP0t**
Fixed some typos and improved readability

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,35 @@
-Ghub.el — Minuscule client library for the Github API
+Ghub.el — Miniscule client library for the GitHub API
```
Comment (tarsius) [inline README.md]: Both spellings exist, but "minuscule" is more common and traditional (its how the Romans spelled it).

File: README.md [hunk 2]
```diff
@@ -1,35 +1,35 @@
...
 =====================================================
 
-Ghub is a library that provides basic support for using the Github
+Ghub is a library that provides basic support for using the GitHub
```
Comment (tarsius) [inline README.md]: I know that the company spells its name that way, but I prefer not to have any capital letters appear in the middle of words.

File: README.md [hunk 3]
```diff
@@ -1,35 +1,35 @@
...
 REST (v3) and GraphQL (v4) APIs from Emacs packages.  It abstracts
 access to API resources using only a handful of functions that are
 not resource-specific.
 
-It also handles the creation, storage and use of access tokens using a
-setup wizard, to make it easier for users to get started and to reduce
-the support burden imposed on package maintainers.  It also comes with
+It also handles the creation, storage, and use of access tokens by using a
```
Comment (tarsius) [inline README.md]: I did some research about this some time ago and decided to go with the Oxford comma even though it feels very strange (we don't use that in German). Two or three years later it still feels very strange and I have decided to go back to the way that feels natural to me (and also is used by many native English speakers).

I think `by` is optional here.

File: README.md [hunk 4]
```diff
@@ -1,35 +1,35 @@
...
+setup wizard to make it easier for users to get started and to reduce
+the support burden imposed on package maintainers and comes with
```
Comment (tarsius) [inline README.md]: I agree that two `It also` is bad, and fixed it by replacing the first one with `Ghub`.

File: README.md [hunk 5]
```diff
@@ -1,35 +1,35 @@
...
 a comprehensive manual to address the cases when things don't just
 work as expected.
 
 Ghub is intentionally limited to only provide these two essential
-features — basic request functions and guided setup — to avoid being
+features—basic request functions and guided setup—to avoid being
```
Comment (tarsius) [inline README.md]: I believe that both styles are used and maybe even that the one that you are proposing is the one people use you know what they are doing. Again it looks really weird to me and after the Oxford comma experience I don't feel like doing a similar experiment again and just stick to what I like.

When using a proportional font the "connected" style actually looks okay, but this text also appears in other places where a monospace font is used and then the connected style looks bad.

**Comment by tarsius** [comment]
Thanks for your effort though!

File: README.md (hunk 3 above)
Comment (vermiculus) [reply README.md]: > many native English speakers

Never assume the majority pays attention to this stuff, though :-) I think everyone knows that oxford-comma is *correct*; it's just that not everyone *cares*. Such is language.

**Pull request closed without merging**